forgerock-auth/src/main/java/org/forgerock/android/auth/OAuth2Client.java

```diff
@@ -202,6 +216,35 @@ public void onResponse(@NotNull Call call, @NotNull Response response) {
         }
     }
 
+    public void endSession(@NonNull String idToken, FRListener<Void> listener) {
```

Comment:
Missing Javadoc. Same comment may apply for others methods in this class.

Reply:
Updated.